save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ExecutionController.kt

```diff
@@ -253,6 +254,21 @@ class ExecutionController(private val executionService: ExecutionService,
             }
     }
 
+    @GetMapping(path = ["/api/$v1/getTestRootPathByExecutionId"])
+    @Transactional
+    fun getTestRootPathByExecutionId(@RequestParam id: Long, authentication: Authentication): Mono<String> =
+        Mono.justOrEmpty(executionService.findExecution(id))
+            .switchIfEmpty() {
+                Mono.error(ResponseStatusException(HttpStatus.NO_CONTENT))
```

Comment:
Why `204`? Maybe, `NOT_FOUND`?

Reply:
Yeah, good idea. My intention was to explicitly mark that we don't have content in the response